tools/provision/formula/Abstract/abstract-osquery-formula.rb

```diff
@@ -158,7 +158,7 @@ def setup_inject
     append "CFLAGS", "-fPIC -DNDEBUG -Os -march=core2"
     append "CXXFLAGS", "-fPIC -DNDEBUG -Os -march=core2"
 
-    if ENV["CC"].to_s.include?("clang") and !["librpm", "python"].include?(self.name)
+    if ENV["CC"].to_s.include?("clang") and !["librpm", "python", "librdkafka"].include?(self.name)
```

Comment:
I'll have to investigate this need to an exception later on, this is concerning. :(

Reply:
haha yea. However I'm not sure I quite understand why we want `-fvisibility=hidden -fvisibility-inlines-hidden` in the first place?

Reply:
We may re-evaluate in the future but for now it is recommended and saving us quite a bit of binary size.